### PR TITLE
Use the correct withdrawal

### PR DESCRIPTION
**Pull request by terencechain**
Withdrawal is not `uin64`, it's an [composite object](https://github.com/ethereum/consensus-specs/blob/dev/specs/capella/beacon-chain.md#withdrawal)

### CODE REVIEW

File: proto/prysm/v1alpha1/validator.proto [hunk 1]
```diff
@@ -676,21 +676,21 @@ message ValidatorCapella {
...
+    // https://github.com/ethereum/consensus-specs/blob/v1.1.10/specs/phase0/beacon-chain.md#constants
     uint64 withdrawable_epoch = 8 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/consensus-types/primitives.Epoch"];
 
     // New in Capella.
     //
     // Epoch when the validator's funds were withdrawn.
     // This field is FAR_FUTURE_EPOCH if the validator's withdrawal has not been processed.
     // FAR_FUTURE_EPOCH is a constant defined by the official Ethereum Beacon Chain specification:
-    // https://github.com/ethereum/consensus-specs/blob/v0.9.2/specs/core/0_beacon-chain.md#constants
+    // https://github.com/ethereum/consensus-specs/blob/v1.1.10/specs/phase0/beacon-chain.md#constants
```
Comment (terencechain) [inline proto/prysm/v1alpha1/validator.proto:693]: cosmetic, I prefer to use the latest stable tag upon new addition

**Review by james-prysm — APPROVED** [review]
LGTM

File: proto/engine/v1/execution_engine.proto [hunk 2]
```diff
@@ -125,3 +125,9 @@ message ForkchoiceState {
 	bytes safe_block_hash      = 2 [(ethereum.eth.ext.ssz_size) = "32"];
 	bytes finalized_block_hash = 3 [(ethereum.eth.ext.ssz_size) = "32"];
 }
+
+message Withdrawal {
+	uint64 index = 1;
+	bytes address = 2 [(ethereum.eth.ext.ssz_size) = "20"];
+	uint64 amount = 3;
+}
```
Comment (rkapka) [inline proto/engine/v1/execution_engine.proto:133]: Withdrawal is already defined here: https://github.com/prysmaticlabs/prysm/blob/78c55019e629885622f565c3e074e141d6b9f50e/proto/prysm/v1alpha1/withdrawals.proto#L28

**Review by rkapka — CHANGES_REQUESTED** [review]
Please use existing `Withdrawal` object.

**Comment by terencechain** [comment]
> [proto/engine/v1/execution_engine.proto](/prysmaticlabs/prysm/pull/11123/files/ac41950dde21d438991601644aad9cdfc33cbede#diff-7324a5b0ced1a588a3ed1e4884affe5241f19a98dd22daa23badfa29333933d2)

I think we should move withdrawal from `prysm/proto/prysm/v1alpha1/withdrawals.proto` to `proto/engine/v1/withdrawals.proto` to avoid confusion

File: proto/engine/v1/execution_engine.proto (hunk 2 above)
Comment (terencechain) [reply proto/engine/v1/execution_engine.proto:133]: I think we should move withdrawal from prysm/proto/prysm/v1alpha1/withdrawals.proto to proto/engine/v1/withdrawals.proto to live next to implementation. Any opposition to this?

**Comment by terencechain** [comment]
@rkapka ready again ser

**Review by rkapka — APPROVED** [review]
ty

**Pull request merged**